packages/x-charts/src/ChartsTooltip/DefaultChartsAxisTooltipContent.tsx

```diff
@@ -18,7 +18,21 @@ function DefaultChartsAxisTooltipContent(props: ChartsAxisContentProps) {
   if (dataIndex == null) {
     return null;
   }
-  const axisFormatter = axis.valueFormatter ?? ((v) => v.toLocaleString());
+
+  const utcFormatter = (v: any): string => {
+    try {
+      const userProvidedDate = new Date(v);
+      const utcDate = new Date(userProvidedDate.toISOString());
+      return utcDate.toUTCString();
+    } catch (e) {
+      return v.toLocaleString();
+    }
+  };
```

Comment:
Not sure about this `try` / `catch`

You could just test if it's a date object. Then return either the `toUTCstring` otherwise `toLocaleString()`

https://masteringjs.io/tutorials/fundamentals/typeof-date

Reply:
I assume Day.js and Moment.js objects have different APIs and are not compatible with instanceof Date